apps/test/unit/templates/rubrics/RubricContainerTest.jsx

```diff
@@ -1,6 +1,6 @@
 import React from 'react';
 import {expect} from '../../../util/reconfiguredChai';
-import {shallow} from 'enzyme';
+import {mount, shallow} from 'enzyme';
```

Comment:
this reminds me that we should put our heads together regarding a technical proposal to start using React Testing Library. We tried it out in the curriculum catalog and found no real blockers

Reply:
Yes! I read Meg's doc that they wrote before they left and it seems like it was pretty easy to use. I'm slightly regretting not using it for this project now, especially as I'm needing to use `mount` quite a bit anyway